TCPEchoServer/Socket.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include <WinSock2.h>
+#include <string>
+#include <vector>
+class __declspec(dllexport) Socket
```

Comment:
Please use the SOCKETDLL_API definition instead of writing __declspec(dllexport)

Reply:
It does not work, called a lot of link errors

Reply:
It does not work, called a lot of link errors